frontend/src/Components/DropdownComponent.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import Style from "style-it";
+import InputComponent from "./InputComponent"
+
+
+class DropdownComponent extends Component {
```

Comment:
A better name for this component is 'Select' component since it is mirroring the 'select' (html tag) component